docs/get_start.rst

```diff
@@ -169,3 +169,109 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
         Network Adapter: Bridged Adapter
         Promiscuous Mode: on
 
+Methodology for booting virtual nodes
+------------------------------------------------
+
+There are three drives for booting virtual nodes, which are network (pxe), disk and cdrom. We can modify the ``boot_order`` in YAML configuration file (The default configuration for OVA is `default.yml <https://github.com/InfraSIM/tools/blob/master/packer/scripts/infrasim.yml>`_, and the default path is ``~/.infrasim/.node_map/default.yml``) or send ipmitool command to choose the drive for booting.
+
+Booting from network
+~~~~~~~~~~~~~~~~~~~~~
+
+You can set the ``boot_order`` as ``n`` before the node start::
+
+    set the boot_order = n in the YAML configuration file
```

Comment:
This is not a statement in yml file and command on console. suggest to refine to:

You can set the ``boot_order`` as ``n`` in yml configuration file
```
boot_order: n
```
Then start node